### PR TITLE
refactor the component library

### PR DESCRIPTION
**Pull request by sushmita12345**
Refactor the Component Library

Screenshot 2022-03-24 at 7 55 34 PM

### CODE REVIEW

**Review by saurabhchirde — APPROVED** [review]
Couldn't find any issue. 
Looks good to merge.

File: assets/css/reset.css [hunk 1]
```diff
@@ -50,14 +51,16 @@ body {
   --orchid-shade: #da70d6;
   --light-orchid: #bb86fc;
   --primary-color1: #2b0e64;
-  --secondary-color1: #a2ffb9;
+  /* --secondary-color1: #a2ffb9; */
```
Comment (vaishnavipaliya11) [inline assets/css/reset.css:54]: remove commented lines from code

File: assets/css/reset.css [hunk 2]
```diff
@@ -50,14 +51,16 @@ body {
...
+  --secondary-color1: palevioletred;
   --tertiary-color1: #5e17eb;
 }
 
 .darkmode {
   /* --off-white: #32174d; */
   --off-white: #28282b;
-  --secondary-color: #a2ffb9;
+  /* --secondary-color: #a2ffb9; */
```
Comment (vaishnavipaliya11) [inline assets/css/reset.css:62]: remove commented lines of code

**Review by vaishnavipaliya11 — APPROVED** [review]
Remove commented lines of code. Everything else looks clean and good you can go ahead.

File: assets/css/reset.css (hunk 2 above)
Comment (sushmita12345) [reply assets/css/reset.css:62]: Ok thanks, I'll do that

**Pull request merged**